src/module.rs

```diff
@@ -515,6 +522,7 @@ impl ModuleInstance {
 	pub fn new<'m, I: ImportResolver>(
 		loaded_module: &'m Module,
 		imports: &I,
+		config: &InterpreterConfig
```

Comment:
maybe leave `new` function as it is now and make another constructor `with_limit` to limit public api changes?

Reply:
I totally agree that changing the `new` function will affect all the callers... Adding a new `with_limits` sounds better, but probably it should be `with_limit_and_externals`, the current API in this struct doesn't quite allow chain API design. Another idea is to use some kind of builder pattern.

Reply:
Yeah, great idea about builder pattern!